src/frequenz/channels/_select.py

```diff
@@ -284,7 +284,7 @@ def selected_from(
 
 
 class SelectError(BaseException):
-    """A base exception for [`select()`][frequenz.channels.select].
+    """An error originated in a [`select()`][frequenz.channels.select] operation.
```

Comment:
Because `select` is an operation and not an object like a `Receiver`, I think "happened during" might be better:

```suggestion
    """An error that happened during a [`select()`][frequenz.channels.select] operation.